src/main/java/at/ac/fhcampuswien/fhmdb/controller/AbstractViewController.java

```diff
@@ -0,0 +1,145 @@
+package at.ac.fhcampuswien.fhmdb.controller;
+
+import at.ac.fhcampuswien.fhmdb.dao.WatchlistRepository;
+import at.ac.fhcampuswien.fhmdb.exceptions.DatabaseException;
+import at.ac.fhcampuswien.fhmdb.handler.ClickEventHandler;
+import at.ac.fhcampuswien.fhmdb.models.Movie;
+import at.ac.fhcampuswien.fhmdb.ui.MovieCell;
+import at.ac.fhcampuswien.fhmdb.ui.SVG;
+import com.jfoenix.controls.JFXButton;
+import com.jfoenix.controls.JFXListView;
+import javafx.collections.FXCollections;
+import javafx.collections.ObservableList;
+import javafx.fxml.FXML;
+import javafx.fxml.FXMLLoader;
+import javafx.scene.Scene;
+import javafx.scene.control.Alert;
+import javafx.scene.control.ButtonBar;
+import javafx.scene.control.ButtonType;
+import javafx.scene.control.Dialog;
+import javafx.scene.layout.StackPane;
+import javafx.scene.layout.VBox;
+import javafx.scene.paint.Color;
+import javafx.scene.shape.SVGPath;
+import javafx.stage.Stage;
+
+import java.io.IOException;
+import java.util.Comparator;
+import java.util.List;
+
+public abstract class AbstractViewController {
+
+    @FXML
+    protected JFXListView<Movie> movieListView = new JFXListView<>();
+
+    @FXML
+    protected VBox navigationMenu;
+
+    @FXML
+    public JFXButton navigationButton;
+
+    @FXML
+    public JFXButton aboutButton;
+
+    protected SVGPath cross = new SVGPath();
+
+    protected SVGPath burger = new SVGPath();
+    protected final ObservableList<Movie> movies = FXCollections.observableArrayList();
+
+    protected ClickEventHandler<Movie> onWatchlistButtonClicked;
+
+    protected WatchlistRepository repository;
+
+    public static final String NO_DB_CONNECTION_AVAILABLE = "No database connection available";
+
```

Comment:
### Remove Button not shown in the Movie-Cell

Add the following code here:
`protected boolean isWatchlistCell = false;`